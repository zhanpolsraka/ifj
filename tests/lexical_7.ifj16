//1
class Main{
	static void run(){
		int x = 2a5;
	}
}