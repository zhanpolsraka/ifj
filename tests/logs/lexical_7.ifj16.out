
Lexical error! Unknown lexem "2a"!

